includes/views/advanced-numbers.php

```diff
@@ -17,7 +17,11 @@
 			}
 		?>
 	</ul>
-	<?php if ( ! empty( $selected_table_name ) && ! empty( $number_store_tables[ $selected_table_name ] ) ) : ?>
+	<?php if ( ! empty( $document_type ) && 'invoice' !== $document_type && in_array( $document_type, $invoice_number_store_doc_types ) ) : ?>
+		<div class="notice notice-warning inline">
+			<p><?php _e( 'Using invoice number!', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
```

Comment:
I would change this to: `This document is currently using the main invoice number sequence.`